wrappers/javascript/anoncreds-react-native/cpp/turboModuleUtility.cpp

```diff
@@ -71,22 +71,6 @@ jsi::Value createReturnValue(jsi::Runtime &rt, ErrorCode code, int8_t *value) {
   return object;
 }
 
-template <>
```

Comment:
Only relevant change, rest is just formatting.

Reply:
I know for android i had to also make one more generic in type. Just commenting this one didn't work (see my example code snippet in slack). 

I'll test in a bit